packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -129,26 +128,62 @@ test.serial('writes fee config to vstorage', async t => {
   await documentStorageSchema(t, storage, doc);
 });
 
-test.serial('writes status updates to vstorage', async t => {
-  const { walletFactoryDriver: wd, storage } = t.context;
-  const wallet = await wd.provideSmartWallet(
-    'agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78',
+test.serial('makes usdc advance', async t => {
+  const { walletFactoryDriver: wd, storage, agoricNamesRemotes } = t.context;
+  const oracles = await Promise.all([
+    wd.provideSmartWallet('agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8'),
+    wd.provideSmartWallet('agoric1krunjcqfrf7la48zrvdfeeqtls5r00ep68mzkr'),
+    wd.provideSmartWallet('agoric1n4fcxsnkxe4gj6e24naec99hzmc4pjfdccy5nj'),
+  ]);
+  await Promise.all(
+    oracles.map(wallet =>
+      wallet.sendOffer({
+        id: 'claim-oracle-invitation',
+        invitationSpec: {
+          source: 'purse',
+          instance: agoricNamesRemotes.instance.fastUsdc,
+          description: 'oracle operator invitation',
+        },
+        proposal: {},
+      }),
+    ),
   );
-  const submitMockEvidence = (mockEvidence: CctpTxEvidence) =>
-    wallet.sendOffer({
-      id: 'submit-mock-evidence',
-      invitationSpec: {
-        source: 'agoricContract',
-        instancePath: ['fastUsdc'],
-        callPipe: [['makeTestPushInvitation', [mockEvidence]]],
-      },
-      proposal: {},
-    });
-  const mockEvidence1 = MockCctpTxEvidences.AGORIC_PLUS_OSMO();
-  const mockEvidence2 = MockCctpTxEvidences.AGORIC_PLUS_DYDX();
 
-  await submitMockEvidence(mockEvidence1);
-  await submitMockEvidence(mockEvidence2);
+  // @ts-expect-error it doesnt recognize usdc as a Brand type
+  const usdc = agoricNamesRemotes.vbankAsset.USDC.brand as Brand<'nat'>;
+  await oracles[0].sendOffer({
+    id: 'deposit-lp-0',
+    invitationSpec: {
+      source: 'agoricContract',
+      instancePath: ['fastUsdc'],
+      callPipe: [['makeDepositInvitation', []]],
+    },
+    proposal: {
+      give: {
+        USDC: { brand: usdc, value: 150_000_000n },
+      },
+    },
+  });
+  await eventLoopIteration();
+
+  const evidence = MockCctpTxEvidences.AGORIC_PLUS_OSMO();
+  // TODO - start counting computrons
+  await Promise.all(
+    oracles.map(wallet =>
+      wallet.sendOffer({
+        id: 'submit-mock-evidence-osmo',
+        invitationSpec: {
+          source: 'continuing',
+          previousOffer: 'claim-oracle-invitation',
+          invitationMakerName: 'SubmitEvidence',
+          invitationArgs: [evidence],
+        },
+        proposal: {},
+      }),
+    ),
+  );
+  await eventLoopIteration();
```

Comment:
Worth simulating the acknowledgement from the vtransfer bridge?

```js
 await runInbound(
    BridgeId.VTRANSFER,
    buildVTransferEvent({
    ...
    }),
  )
```

Reply:
Hmm good point, I think this would happen in a different block, so maybe measure it separately, but should be counted nonetheless. I can follow up with this.